### ISSUE TITLE
Values from API differs from CSV from EA Portal

### ISSUE THREAD

**Issue by mmilhomem** [Microsoft/AzureUsageAndBillingPortal#67]
Hi,
   I have a customer (EA) using this solution. He is having an issue where the consumed quantity and the cost does not match the ones that he downloads from the ea portal (CSV) now he does not know witch source is the truth.

**Comment by tdjastrzebski** [comment]
Yeah, this was mystery to me as well. It seems like there are bugs in rate cards or at least some inconsistencies. Occasionally we were getting nonexistent rates in records. That is why we ended up using a similar approach but based on EA CSV data :/
